<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signup</class>
 <widget class="QMainWindow" name="signup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>signup</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>400</width>
      <height>480</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>B Titr</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Register</string>
    </property>
    <widget class="QLineEdit" name="UserName_lineEdit">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>80</y>
       <width>250</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="Password_lineEdit">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>130</y>
       <width>250</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="Phone_lineEdit">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>180</y>
       <width>250</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="Email_lineEdit">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>230</y>
       <width>250</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="captcha_groupBox">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>280</y>
       <width>250</width>
       <height>100</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>B Titr</family>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>captcha</string>
     </property>
     <widget class="QPushButton" name="changCaptcha_pushButton">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>35</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>chang</string>
      </property>
     </widget>
     <widget class="QLabel" name="captcha_label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>150</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>B Titr</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="captcha_lineEdit">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>80</y>
        <width>120</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="AddPlayer_pushButton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>420</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>B Zar</family>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>480</y>
      <width>400</width>
      <height>100</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>25</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>B Titr</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Lets Play...</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
